### ISSUE TITLE
Send from cmd line to slack works, not from Zabbix to slack.

### ISSUE THREAD

**Issue by GPegel** [ericoc/zabbix-slack-alertscript#15]
Thanks for this beautiful script. But I have an issue. Everything seems to work perfectly. I'm able to send test messages to my slack channels from the command line on the server where Zabbix is installed. But what doesn't work is sending alerts from zabbix to slack. Zabbix says it sends the message and everything is oke but the message does not receive in Slack.

Do the slack scripts need some proper user rights or a CHMOD or what ever? Thanks in advance

**Comment by ericoc** [comment]
As per the [README](https://github.com/ericoc/zabbix-slack-alertscript#installation):

> the slack.sh script needs to be placed in the AlertScriptsPath directory that is specified within the Zabbix servers' configuration file (zabbix_server.conf) and must be executable by the user running the zabbix_server binary (usually "zabbix")

**Comment by GPegel** [comment]
The script is placed in the correct directory and the script is executable. I'm not sure if it's executable by the zabbix user. I will check and come back with results. Thank you for your quick reply.

**Comment by GPegel** [comment]
I fixed it! I had to add this to "visudo" to get it working: "Defaults:zabbix !requiretty" Now it works perfect!

**Comment by bughatti** [comment]
Hello and thank you for this script.  I am curious if I need to ask for help in here or on the Zabbix forums.  I am having the same issue as above.  I can walk an alert via command line but zaabbix is not sending anything else.  I am currently running Ubuntu 16.04 Desktop.  The slack.sh script was originally owned by a different admin user in Zabbix, but I changed it to zabbix with sudo chown zabbix:zabbix /usr/lib/zabbix/alertscripts/slack.sh I also followed the instructions step by step for the media type,user and actions.  I have also tried the above to put Defaults:zabbix !requiretty in the visudo file.  I create a problem for a device and I get a report on my dashboard saying the device is down but no alert shows up in slack.

Any help would be greatly appreciated!
